src/components/App/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Pretty sure most of this is cruft left over from the default app that create-react-app gives you.